Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics/MicroGraphics/Driver/Graphics.MicroGraphics/MicroGraphics.csproj

```diff
@@ -17,6 +17,9 @@
   </PropertyGroup>
   <ItemGroup>
     <None Include="..\..\..\..\..\icon.png" Pack="true" PackagePath="" />
-    <ProjectReference Include="..\..\..\..\..\Meadow.Foundation.Core\Meadow.Foundation.Core.csproj" />
```

Comment:
I think the goal is to leave the local references in place for the develop branch - but we really need to setup the nuget/ref configs - for now can you skip this change?